### ISSUE TITLE
Do not output blank lines.

### ISSUE THREAD

**Issue by mondy** [hairyhenderson/gomplate#1111]
I'm using gomplate with Windows PowerShell.
Save the following code as test.ps1 and run it.
A blank line is printed between foo and bar. I don't want to output blank lines.

```powershell
echo foo
gomplate -f  test.ps1 | Set-Content -Encoding Unicode output
echo bar
```

Output:
```
foo

bar
```

Save the following code as test.ps1 and run it.
No blank lines are printed between foo and bar.
Why is this difference? 

```powershell
echo foo
gomplate -h | Set-Content -Encoding Unicode output
echo bar
```

Output:
```
foo
bar
```

# Environments
* OS: Windows 10
* Shell: PowerShell
* gomplate version 3.9.0

**Comment by hairyhenderson** [comment]
Hi @mondy, I believe this is a duplicate of #1075 - it's fixed and will be part of the 3.10.0 release

**Comment by mondy** [comment]
Thank you for reply. 
I overlooked this issue.
I'm looking forward to the release of 3.10.0.
